### PR TITLE
scaladex release note

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: blog/_posts/2016-08-08-the-scala-library-index-is-released.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+![Background](/resources/img/blog/scaladex/head-project-background.png)
+
+The Scala Center team is really proud to announce the release of <a href="https://index.scala-lang.org/" target="_blank">The Scala Package Index</a> or Scaladex for short.
+
+## Why do we need an index for Scala ?
+
+With this new tool, a developer can query more than 100 000 scala library releases. We group those releases into projects, artifact, targets and versions. The Index is similar to a DNS (Domain Name Server). With a project or artifact name, you can find the exact release to paste into your build.
+
+The Index is also similar to a catalog. You will find new projects or new releases for existing projects. Authors can their project, such as adding keywords, to improve the search experience.
```
Comment (julienrf) [inline blog/_posts/2016-08-08-the-scala-library-index-is-released.md]: “Authors can their project”. There is a missing word.

File: blog/_posts/2016-08-08-the-scala-library-index-is-released.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## How does it works ?
```
Comment (julienrf) [inline blog/_posts/2016-08-08-the-scala-library-index-is-released.md]: “work**s**” -> “work”

File: blog/_posts/2016-08-08-the-scala-library-index-is-released.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+![Background](/resources/img/blog/scaladex/head-project-background.png)
+
+The Scala Center team is really proud to announce the release of <a href="https://index.scala-lang.org/" target="_blank">The Scala Package Index</a> or Scaladex for short. We would like to thanks our open source contributors: <a href="https://github.com/Scyks" target="_blank">Ronald Marske</a> and the <a href="http://www.47deg.com/" target="_blank">47 Degrees</a>.
+
+<iframe width="610" height="315" src="https://www.youtube.com/embed/TBoJivIJsbU" frameborder="0" allowfullscreen></iframe>
+
+## Why do we need an index for Scala ?
+
+With this new tool, a developer can query more than 100 000 scala library releases. We group those releases into projects, artifact, targets and versions. The Index is similar to a DNS (Domain Name Server). With a project or artifact name, you can find the exact release to paste into your build.
```
Comment (jvican) [inline blog/_posts/2016-08-08-the-scala-library-index-is-released.md]: s/artifact/artifacts :smile:

File: blog/_posts/2016-08-08-the-scala-library-index-is-released.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+When we have all the release information we can group them by the GitHub repository they provide. If you forgot to include this information you can <a href="https://github.com/scalacenter/scaladex-data/edit/master/claims.json" target="_blank">claim your project</a> via a pull request.
+
+Scaladex has the advantage of performing well without manual intervention. However with the help of library owners we can improve the quality of our data. We kindly invite you to login to github and start editing your projects.
+
+## What's next ?
+
+The Scala Center has a limited ammount of ressource and an impressive ammount of upcomming project. For this reason, we would like to work with the scala community to continue the development of this project. We have identified a [set of task](https://github.com/scalacenter/scaladex/issues?q=is%3Aopen+is%3Aissue+label%3Av2) that could be resolve by the community. We are also open to new ideas to improve the index.
```
Comment (jvican) [inline blog/_posts/2016-08-08-the-scala-library-index-is-released.md]: s/ressource/resource :smile:

**Comment by SethTisue** [comment]
Jorge found some typos/spelling problems, but there are more — a spellchecker would catch most of them. also capitalization: Scala not scala, GitHub not github, etc, and problems with singular/plural. Guillaume, if you want me to go through and fix it, make me a contributor on your fork?

**Comment by MasseGuillaume** [comment]
@SethTisue I will rerun the spellchecker ;-)

**Comment by SethTisue** [comment]
you already fixed most of what I had spotted before, but I made a few additional tweaks.

**Comment by heathermiller** [comment]
I added more detail about Scaladex and renamed the file to reflect the actual publishing date (today, August 9) and I massaged the title a bit.

**Pull request merged**